src/stytch-provider.ts

```diff
@@ -44,18 +50,150 @@ function StytchProvider(this: any, options: StytchProviderOptions) {
             action: async function(this: any, entize: any, msg: any) {
               let q = msg.q || {}
 
-              // NOTE: throws on error
-              let res = await this.shared.sdk.users.search(q)
-              let list = res.results.map((data: any) => entize(data, {
+	      q = Object.keys(q).reduce((acc: any, key: any) => {
+		if(key.endsWith('$')) {
+		  key = key.replace(/.$/, '')
+		  acc[key] = q[key+'$']
+		} else acc[key] = q[key]
+		return acc
+	      }, {})
+	      let res: any = null
+
+	      try {
+                res = await this.shared.sdk.users.search(q)
+	      }catch(err_res) {
+		res = err_res
+	      }
+	      check_status.call(this, res)
+
+	      let list = res.results.map((data: any) => entize(data, {
                 field: {
                   id: { src: 'user_id' }
                 }
-              }))
+	      })
+
+	      )
               return list
             }
           },
+
+	  load: {
+	    action: async function(this: any, entize: any, msg: any) {
+	      let id = msg?.q?.id
+	      let res: any = null
+
+	      id == null ? this.fail('invalid id') : null
+
+
+	      try {
+                res = await this.shared.sdk.users.get(id)
+	      }catch(err_res) {
+		res = err_res
+	      }
+	      check_status.call(this, res)
+
+	      // TODO: something like: (entize({'res': res} as ProviderRes)) for more structure?
+	      return entize(res, {
+                field: {
+                  id: { src: 'user_id' }
+                }
+              })
+
+	    }
+	  },
+	  
+          save: {
+	    action: async function(this: any, entize: any, msg: any) {
+	      let ent: any = msg.ent
+	      let id = ent.id
+	      let user: any = ent['user'] || {}
+	      let params: Array<any> = []
+	      let api_call: string
+
+	      let res: any = null
+
+	      api_call = id == null ?
```

Comment:
```
let id:any = 1
console.log(id = null ? 'id is null' : 'id:'+id)
console.log('oops id=',id)
```

<img width="329" alt="Screenshot 2023-05-03 at 14 59 51" src="https://user-images.githubusercontent.com/138608/235938469-8d7ba8e4-17da-4b17-973a-c67bddde788b.png">

Reply:
thus: `api_call = id == null ?` should be `api_call = (null == id) ?`

Reply:
got it! Done!

Reply:
got it! done!